uddl_test/uddl_test_formatting/universal-data-definition-language-formatter.test.ts

```diff
@@ -46,14 +43,84 @@ describe("Universal Data Definition Language Formatter", () => {
         
         };
         
+    }
+    
+}`,
+    });
+  });
+
+  it("should format cdm with conceptual entity", async () => {
+    await universalDataDefinitionLanguageFormatting({
+      before:
+        'dm PPT "Base data structures to support People, Places and Things"{cdm Conceptual "Need to start at Conceptual Level"{centity AddressableEntity "Any entity that is addressable in some way" {};}}',
```

Comment:
In your current tests, all the 'before' text is on a single line. You should also have the same tests where the 'before' text has each token on a separate line (you can do this with a find/replace that replaces blanks the `before` text with CRs).

Reply:
Noted